tests/test_sdss_reconstruct.py

```diff
@@ -0,0 +1,36 @@
+import torch
+from hydra.utils import instantiate
+
+from bliss.predict import predict, prepare_image
+
+
+class TestSdssReconstrust:
+    def test_sdss_reconstrst(self, cfg):
+        sdss = instantiate(cfg.predict.dataset)
+        true_img = sdss[0]["image"][:, 160:320, 160:320]
+        true_bg = sdss[0]["background"][:, 160:320, 160:320]
+        prepare_img = prepare_image(true_img, cfg.predict.device)
+        prepare_bg = prepare_image(true_bg, cfg.predict.device)
+        est_full = predict(cfg, prepare_img, prepare_bg)
+        est_tile = est_full.to_tile_params(cfg.encoder.tile_slen, cfg.simulator.prior.max_sources)
```

Comment:
Perhaps `predict_sdss(cfg)` could be called here too? If so, we'd need the config file include the ranges 160:320 and 160:320.